packages/svelte/tests/runtime-runes/samples/await-no-catch-error/main.svelte

```diff
@@ -0,0 +1,12 @@
+<script>
+	const promise = new Promise((res, rej) => {
+		rej(new Error('Test'));
+	})
```

Comment:
AFAIK this should also do it
```suggestion
	const promise = Promise.reject('Test');
```